db/tidb/sequential_test.go

```diff
@@ -1,45 +1,46 @@
 package tidb
```

Comment:
So where is this testing?

Reply:
some workloads aren't used at present, so I commented these code.